routes/results.js

```diff
@@ -1,8 +1,12 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 
 router.get('/', function(req, res, next) {
 	res.render('results');
 });
 
+router.get('/:name/:level', function(req, res, next) {
```

Comment:
why does market level need to be specified? shouldn't the app tell users the market's level?

Reply:
The app does tell users their market level.  The results page is just a quick place you are redirected to so that you can immediately see what you were ranked.  The results page could also pull from Firebase, but that seems unnecessary since you already have the market level as it was just calculated.

This way you save having to pull the info again and the user can immediately see what their market was ranked without having to search for it in the markets page.

Reply:
What's the origin of the redirect?

Reply:
The origin is submit-assess.  When the submission happens it posts it to the database and redirects to this page to show that it worked and that the market was ranked with the given ranking.